src/Lean/Elab/Print.lean

```diff
@@ -65,8 +65,36 @@ private def printInduct (id : Name) (levelParams : List Name) (numParams : Nat)
     m := m ++ Format.line ++ ctor ++ " : " ++ cinfo.type
   logInfo m
 
+open Meta in
+private def printStructOrClass (id : Name) (levelParams : List Name) (numParams : Nat) (type : Expr)
```

Comment:
Perhaps just `printStructure`? (No need to abbreviate `Structure`, unless it's being done consistently elsewhere.)